### ISSUE TITLE
Improve implementation of limits on outputs

### ISSUE THREAD

**Issue by jeremie-fouquet** [spinicist/QUIT#42]
The origin of this issue on my side was that our data generates MTsat values higher than 10%, which get clamped [here](https://github.com/spinicist/QUIT/blob/58f9f40bec51b6af9c4219237511fa1166823420/Source/MT/MTSatModel.cpp#L22)

This might call for a general solution, which could be one of:
- Revise and relax the limits on the outputs 
- Include the limits as parameters

Obviously, the first option is less involved than the second. Let us know what you prefer.

Either way, I would also include the limits in the documentation to make them more transparent to the users. 

@gdevenyi

**Comment by spinicist** [comment]
It's fine to add another member to the `MTSatModel` that defines the upper limit and pass that in as an argument, as you did for the small angle approximation.

However - are you sure you are getting 10% MTSat? IIRC MTSat is defined as the reduction in water Mz by the MT pulse per TR. For typical clinical scan parameters getting that to more than a few percent is _hard_ and would involve very long pulses.

**Comment by jeremie-fouquet** [comment]
Yes, that's a good point. Our data is mouse brain data, which we acquired with a "990°" MT pulse (duration of 6ms). This might be a more intense MT pulse than in a clinical setting.

Although I agree it appears high, I'm pretty sure we get MTsat values >10% (~15%), especially in myelin-rich regions.
